src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
```

Comment:
I would suggest sticking to adding margins either on top or on bottom for the whole styling. If some element positioned on top of the other has bottom margin and the element below has top margin set, the margins won't add up and the bigger margin will be the one positioning the elements. This as you may guess leads to unnecessary confusion.